app/decorators/user_decorator.rb

```diff
@@ -43,7 +43,7 @@ class UserDecorator < ApplicationDecorator
   # @return [Array<UserDecorator::CachedTagByUser>]
   def cached_followed_tags
     cached_tag_attributes = Rails.cache.fetch(
-      "user-#{id}-#{following_tags_count}-#{last_followed_at&.rfc3339}/user_followed_tags",
+      "#{cache_key_with_version}/followed_tags",
```

Comment:
in your PR description you said didn't add "with_version" but you ended up adding it here. Just want to clarify that's your intention.

Reply:
@filleduchaos 

I think you are right, `cache_key_with_version` is based on `updated_at` and `id` as the basis for updating the cache key.

So in this case, any kind of updating about user would change the cache key.

I think I can try another way to refactor this key name

Reply:
Hmmm - just `cache_key` + the `last_followed_at` field should work because then it will be like `cache_key_with_version` but using the last followed time instead of any update time